pkg/util/ready/ready.go

```diff
@@ -100,6 +102,24 @@ func CheckDeploymentIsReady(ctx context.Context, cli appsv1client.DeploymentInte
 	}
 }
 
+// CheckAROOperatorDeploymentIsReady returns a function which polls AROOperator Deployment and
+// returns its readiness
+func CheckAROOperatorDeploymentIsReady(ctx context.Context, cli appsv1client.DeploymentInterface, name string) (bool, error) {
+	d, err := cli.Get(ctx, name, metav1.GetOptions{})
+	switch {
+	case kerrors.IsNotFound(err):
+		return false, nil
+	case err != nil:
+		return false, err
+	}
+
+	if d.Labels["version"] == version.GitCommit {
```

Comment:
@hawkowl this is going to change with #1848 isn't it?  

Because the field will now be in the `OperatorFlags` field within cosmos.

Reply:
This variable is still used after the update. So this should stay transparent.